bitnami/external-dns/templates/deployment.yaml

```diff
@@ -446,14 +451,27 @@ spec:
                   key: infoblox_wapi_password
             {{- end }}
             {{- end }}
-            {{- if or .Values.rfc2136.tsigSecret .Values.rfc2136.secretName }}
+            {{- if or .Values.rfc2136.secretName .Values.rfc2136.tsigSecret (and .Values.rfc2136.kerberosUsername .Values.rfc2136.kerberosPassword) }}
```

Comment:
I'm not sure I like this condition:

```
(and .Values.rfc2136.kerberosUsername .Values.rfc2136.kerberosPassword)
```

If I'm not mistaken, this means users can't add the username and password keys to a secret they own and use it via `rfc2136.secretName`. I feel like that should be an option. What do you think?

Reply:
You're misreading the conditional, which is:

.Values.rfc2136.secretName OR .Values.rfc2136.tsigSecret OR (.Values.rfc2136.kerberosUsername AND .Values.rfc2136.kerberosPassword)

Users can specify the secret name, and that is indeed my exact use case

Reply:
You are right!